frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -154,39 +244,70 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
 /**
  * Get an icon type for the specified cluster property details (type, draft, erp, spp etc)
  */
-export const getMarkerIcon = (feature: ICluster) => {
+export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
   const { propertyTypeId, projectWorkflow, classificationId } = feature?.properties;
-  if (projectWorkflow === Workflows.ERP) {
-    switch (propertyTypeId) {
-      case PropertyTypes.SUBDIVISION:
-        return subdivisionErpIcon;
-      case PropertyTypes.BUILDING:
-        return buildingErpIcon;
-      default:
-        return landErpIcon;
-    }
-  } else if (
-    classificationId === Classifications.SurplusActive ||
-    classificationId === Classifications.SurplusEncumbered
-  ) {
-    switch (propertyTypeId) {
-      case PropertyTypes.BUILDING:
-        return buildingSppIcon;
-      case PropertyTypes.SUBDIVISION:
-        return subdivisionSppIcon;
-      default:
-        return landSppIcon;
-    }
-  } else if (propertyTypeId === PropertyTypes.PARCEL) {
-    return parcelIcon;
-  } else if (propertyTypeId === PropertyTypes.SUBDIVISION) {
-    return subdivisionIcon;
-  } else if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
+  if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
```

Comment:
do we want a draft pin to be highlighted? it's already blinking

Reply:
I don't think so, but I have yet to see any UI for draft marker functionality so who knows?